### PR TITLE
carousel

### PR DESCRIPTION
**Pull request by chefReyMysterio**

### CODE REVIEW

File: dist/components/Carousel/CarouselElements/ArrowSquareGrayOpacity/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/// <reference types="react" />
+export declare const PrevButton: ({ enabled, onClick, }: {
+    enabled: boolean;
+    onClick: () => void;
+}) => JSX.Element;
+export declare const NextButton: ({ enabled, onClick, }: {
+    enabled: boolean;
+    onClick: () => void;
+}) => JSX.Element;
```
Comment (lilChemsHulkHogan) [inline dist/components/Carousel/CarouselElements/ArrowSquareGrayOpacity/index.d.ts:9]: maybe we can use smt like NavButton declaration for these buttons?

Comment (lilChemsHulkHogan) [reply dist/components/Carousel/CarouselElements/ArrowSquareGrayOpacity/index.d.ts:9]: just one)

File: src/components/Carousel/CarouselElements/ArrowSquarePrimary/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, { FC, ReactNode } from "react";
+import styled from "styled-components";
+
+interface ButtonProps {
+  onClick: () => void;
+  children: ReactNode;
+}
```
Comment (lilChemsHulkHogan) [inline src/components/Carousel/CarouselElements/ArrowSquarePrimary/index.tsx]: consider moving this interface to types for reusing in ArrowSquareGrayOpacity/index.tsx etc.

**Review by ghost — APPROVED** [review]

File: src/components/Carousel/CarouselElements/ArrowSquareGrayOpacity/index.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  enabled,
+  onClick,
+}: {
+  enabled: boolean;
+  onClick: () => void;
+}) => (
+  <PrevWrap onClick={onClick} disabled={!enabled}>
+    <ChevronLeftIcon color="white" height="32px" />
+  </PrevWrap>
+);
```
Comment (lilChemsHulkHogan) [inline src/components/Carousel/CarouselElements/ArrowSquareGrayOpacity/index.tsx]: please consider reusable button components, as I can see that these pair are similar

File: src/hooks/useCarousel/useCarousel.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  },
+];
+
+const Item: FC<{ text: string }> = ({ text }) => (
+  <Flex
+    p="24px"
+    width="100%"
+    minWidth="100%"
+    alignItems="center"
+    justifyContent="center"
```
Comment (lilChemsHulkHogan) [inline src/hooks/useCarousel/useCarousel.stories.tsx:32]: nice arrow)

**Review by lilChemsHulkHogan — APPROVED** [review]

File: src/components/Carousel/CarouselElements/ArrowSquareGrayOpacity/index.tsx (hunk 3 above)
Comment (chefReyMysterio) [reply src/components/Carousel/CarouselElements/ArrowSquareGrayOpacity/index.tsx]: +

**Review by devStieg — APPROVED** [review]

File: src/components/Carousel/CarouselElements/ArrowSquarePrimary/index.tsx (hunk 2 above)
Comment (chefReyMysterio) [reply src/components/Carousel/CarouselElements/ArrowSquarePrimary/index.tsx]: +

**Pull request merged**